typescript/sdk/src/router/schemas.ts

```diff
@@ -32,6 +32,13 @@ export const RouterConfigSchema = MailboxClientConfigSchema.merge(
   }),
 );
 
+const DestinationGasDomain = z.string();
+const DestinationGasAmount = z.string(); // This must be a string type to match Ether's type
+export const DestinationGasSchema = z.record(
+  DestinationGasDomain,
+  DestinationGasAmount,
+);
 export const GasRouterConfigSchema = RouterConfigSchema.extend({
   gas: z.number().optional(),
+  destinationGas: DestinationGasSchema.optional(),
```

Comment:
okay now im quite confused

Reply:
the intent was for the local domain's `gas` config to be used on the other origins as destination gas

Reply:
is the idea that this `destinationGas` is always derived?

Reply:
yes `destinationGas` is aways derived.